src/Conversions/Casing/PythonCaseConverter.ts

```diff
@@ -0,0 +1,40 @@
+import { PackageLowerCaseConverter } from "./PackageLowerCaseConverter";
+
+/**
+ * Converts a series of words to PythonCase.
+ */
+export class PythonCaseConverter extends PackageLowerCaseConverter {
```

Comment:
As a non-Python developer, this is pretty vague to me. "PythonCase" sounds like it's referring to `snake_case`. What are your thoughts on changing it to "PythonImportCase"?